packages/libcap-ng/build.sh

```diff
@@ -9,4 +9,6 @@ TERMUX_PKG_SHA256=65b86885b8d873e55c05bd49427fd370d559b26f0c2089ac9194828e6a2fe2
 
 termux_step_pre_configure() {
 	./autogen.sh
+	CFLAGS+=" -I${TERMUX_PREFIX}/include/python3.9"
```

Comment:
We are not building python bindings for this package so this should not be needed (python is not listed as a dependency so the build cannot build them with this build.sh anyways). 

If you ran into the error when building on device, where python is installed, then a better fix would be to add `--without-python` to `TERMUX_PKG_EXTRA_CONFIGURE_ARGS`

Reply:
Im building on server computer.
While building qemu, libcap-ng gives error "Python.h not found"
So is that ok to include python folder ?
Or i do as you said

Reply:
Either disable python with the configure arg, or add it as a dependency and bump the package. Building the python bindings will most likely need some more patches for the cross-compilation to work though. I would disable it